opencti-platform/opencti-graphql/src/migrations/1681460290302-group-initialization.js

```diff
@@ -0,0 +1,31 @@
+import { executionContext, SYSTEM_USER } from '../utils/access';
+import { logApp } from '../config/conf';
+import { findAll, groupEditField } from '../domain/group';
+
+export const up = async (next) => {
+  const context = executionContext('migration');
+  const groupsEdges = await findAll(context, SYSTEM_USER);
+  const groups = groupsEdges.edges.map((n) => n.node);
+  const groupEditionPromises = groups.map((group) => {
+    if (group.auto_new_marking === undefined) {
```

Comment:
It could be simplified

Const updateFields = [];
if (group.auto_new_marking === undefined) {
   { key: 'auto_new_marking', value: ['false'] }
}
if (group.default_assignation === undefined) {   
   { key: 'default_assignation', value: ['false'] }
}

if (updateFields.length > 0) {
   return groupEditField(context, SYSTEM_USER, group.id, updateFields);
}

What do you think ?

Reply:
you're right! it's better this way!